notebooks/official/automl/automl_tabular_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1083 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #48.    # a day or two. The tracking bug is b/230784641.
</pre><p>Bug b/230784641 appears to be resolved, update?</p><p> </p><p>It looks like the variables worker_pool_specs_override and study_spec_override are unused?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/694/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
Fixed.